packages/arb-token-bridge-ui/src/components/common/HeaderAccountPopover.tsx

```diff
@@ -262,9 +236,7 @@ export function HeaderAccountPopover() {
 
             {/* Explorer button */}
             <ExternalLink
-              href={`${getExplorerUrl(
-                currentNetwork?.chainID ?? -1
-              )}/address/${account}`}
+              href={`${getExplorerUrl(chain?.id ?? -1)}/address/${address}`}
```

Comment:
if we don't have `chain?.id` does it make sense to show a link that will trigger a 404 error?